main.go

```diff
@@ -76,16 +69,16 @@ func main() {
 
 	fmt.Print("\033[s\033c")
 	for running {
-		fmt.Print("\033[u")
+		width, height, err := terminal.GetSize(0)
 		ok := webcam.Read(&img)
 		if !ok {
 			fmt.Fprintln(os.Stderr, "Failed to read from camera")
 			return
 		}
+		cv.Resize(img, &img, image.Point{width, (height - 1) * 2}, 0, 0, 1)
```

Comment:
For some reason, my terminal likes to freak out when you try to draw on the bottom line, so the `- 1` here is to stop it doing that. There is probably a way to stop it auto-scrolling which would mean the bottom line would be usable again

Reply:
Fixed in https://github.com/frebib/PixelOnTerminal/commit/0ba7440032160c19bfd1ae0abb368d9f1fa1e902#diff-7ddfb3e035b42cd70649cc33393fe32cR47-R50